index.js

```diff
@@ -65,42 +62,26 @@ if ( protocol === 'https' ) {
 		}
 	}
 
-	const options = {
+	return {
 		key: fs.readFileSync( key ),
 		cert: fs.readFileSync( certificate ),
 	};
-	server = require( 'https' ).createServer( options, app );
-} else {
-	server = require( 'http' ).createServer( app );
 }
 
-// The desktop app runs Calypso in a fork.
-// Let non-forks listen on any host.
-if ( ! process.env.CALYPSO_IS_FORK ) {
-	host = null;
+// Start a development HTTPS server.
+function createServer() {
+	if ( protocol === 'https' ) {
+		return require( 'https' ).createServer( loadSslCert(), app );
+	}
+
+	return require( 'http' ).createServer( app );
 }
 
-server.listen( { port, host }, function() {
+const server = createServer();
```

Comment:
NIT: It would be nice to move this and the following `listen` lines up to L24.

Reply:
@sirreal I.e., put the top-level code first and the helper function definitions at the end?

Reply:
Yes, that seems appealing here, but just a nitpick 🙂